doc/common/abbreviations.md

```diff
@@ -145,27 +157,32 @@
 | RAM            | Random Access Memory                  |
 | RAN            | Radio Access Network                  |
 | RBAC           | Role-bases Access Control             |
-| RFC            | Request for Change                    |
 | RC             | Reference Conformance                 |
+| RFC            | Request for Change                    |
```

Comment:
Wasn't this Request For Comments?

Reply:
@CsatariGergely this was in the original -- I didn't touch it

Reply:
@CsatariGergely if you are OK with the change can you please resolve. Thanks